modules/retail/pages/retail-install-setup.adoc

```diff
@@ -47,52 +74,113 @@ For more information about formulas, see xref:retail:retail-formulas-intro.adoc[
 +
 ----
-retail_branch_init <branch_server_salt_id>
+retail_branch_init <branch_server_minion_id>
+----
++
+This command will configure branch server formulas with default values and for shared networking topology.
+For dedicated network topology run this command:
++
+----
+retail_branch_init <branch_server_minion_id> --dedicated-nic <network_device>
+----
++
+You can customize network information as well, together with custom [systemitem]``branch prefix``. For example:
++
+----
+retail_branch_init <branch_server_minion_id> --dedicated-nic eth1
+                                             --branch-prefix B001
+                                             --server-domain <branch_server_subdomain>
+                                             --branch-ip 192.168.86.1
+                                             --netmask 255.255.255.0
 ----
 +
-This command will configure branch server formulas with recommended values.
 You can use the [command]``retail_branch_init --help`` command for additional options.
+
 . Verify that your changes have been configured correctly by checking the {susemgr} {webui} branch server system formulas.
 . Apply highstate on the branch server.
 You can do this through the {webui}, or by running this command:
 +
 ----
-salt <branch_server_salt_id> state.apply
+salt <branch_server_minion_id> state.apply
 ----
 
+Similar results can be achieved by using mass import command line tool.
 
-== Synchronize Images to the Branch Server
+.Procedure: Configuring Branch Server Formulas With a Mass Import Tool
 
-The OS image you use on the {susemgr} server must be synchronized for use on the branch server.
-You can do this with the Salt [command]``image-sync`` tool.
-
-.Procedure: Synchronizing Images to the Branch Server
-
-. On the {susemgr} server, run this command:
+. Prepare branch specific YAML file:
++
+For example, create branch.yaml file with content:
 +
 ----
-salt <branch_server_salt_id> state.apply image-sync
+branches:
+  <branch_server_minion_id>:
+    branch_prefix: branch1
+    server_name: branchserver1
+    server_domain: example.com
+    nic: eth1
+    dedicated_nic: true
+    configure_firewall: true
+    branch_ip: 192.168.2.1
+    netmask: 255.255.255.0
+    dyn_range:
+        - 192.168.2.10
+        - 192.168.2.250
 ----
-. The image details will be transferred to [path]``/srv/saltboot`` on the branch server.
++
+For more information about mass import tool, see xref:retail:retail-mass-config.adoc[].
+. Import branch information from YAML file to {susemgr}
++
+----
+retail_yaml --from-yaml branch.yaml
+----
+. Verify that your changes have been configured correctly by checking the {susemgr} {webui} branch server system formulas.
+. Apply highstate on the branch server.
 
+[WARNING]
+====
+Both [command]``retail_branch_init`` and [command]``retail_yaml`` commands overwrites existing configuration of specified branch server.
```

Comment:
```suggestion
Both [command]``retail_branch_init`` and [command]``retail_yaml`` commands override existing configuration settings of the specified branch server.
```

Reply:
I don't think override is the correct verd here. That command literally changes the stored configuration

Reply:
I edited the suggestion: changing back to overwrite, but using plural form.

(I'm frankly admit that I treat both verbs as synonyms and usually preferring override. But I can imaging that there are differences.)

Reply:
Override has a lot more usage in the English corpus, because it has a greater variety of uses (https://books.google.com/ngrams/graph?content=override%2C+overwrite&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Coverride%3B%2Cc0%3B.t1%3B%2Coverwrite%3B%2Cc0). AHD has 5 definitions for override as a transitive verb, and 4 as a noun, while overwrite only lists two, because it's really specific to writing, rather than a general sense of going over. In most software contexts, override is more accurate, because we're not just talking about putting new words over old words, but changing settings, which is a greater effect.

Reply:
I don't know, at the end you are the experts here :)
I just put 'override vs overwrite' in the google and the first link for me was https://english.stackexchange.com/questions/88400/when-to-use-override-and-overwrite where they states:

> "Override" is the cancellation of some previous action or decision. "Overwrite" specifically refers to something being written over something previously written. In other words, actions and decisions are overriden. Information is overwritten. Furthermore it is overwritten with new information.

Which is exactly what I'm pointing out. Both `retail_branch_init` and `retail_yaml` destroys previous configuration and write completely new one. Which for me sounds like _"something being written over something previously written"_

Reply:
Yes, but I'm thinking more about the effect of that: it's not the words in the configuration file so much as the settings you are changing. Consider: if you change the settings in the UI (by clicking a button, or selecting from a list), that's overriding, not overwriting. So if you do the same action in a configuration file, it could technically be either. In which case, we should pick the word that applies more generally, which is overriding.

Reply:
Picked the `override` variant :+1: